src/plugins/interactive_setup/server/elasticsearch_service.test.ts

```diff
@@ -537,6 +548,24 @@ some weird+ca/with
         );
       });
 
+      it('fails if host is not Elasticsearch', async () => {
```

Comment:
The unit test looks great! I am wondering if it would be a good idea to add an integration test for this too, so we have some sort of canary if the Elasticsearch product header changes for any reason.

Here's an example of a Jest integration test that uses an actual running ES server (with `startES`):

https://github.com/elastic/kibana/blob/d9873278bb135d78e0c56a2493c4d124123b9a5f/src/core/server/saved_objects/service/lib/integration_tests/repository.test.ts

What do you think?

Reply:
Agreed but there's a separate story for integration tests: https://github.com/elastic/kibana/issues/111336

Reply:
Ah great, we'll tackle it then!